lib/swig/solver/sto_solver.i

```diff
@@ -9,6 +9,7 @@
 %}
 
 %include "model.i"
+%include "prox.i"
```

Comment:
Why do we need to include these here now? Because of templates?

Reply:
swig gives typeerrors without it

Reply:
Ok, swig black magic!